lib/execute-method-map.ts

```diff
@@ -370,6 +370,9 @@ export const executeMethodMap = {
 
   'mobile: getContexts': {
     command: 'mobileGetContexts',
+    params: {
+      optional: ['opts'],
```

Comment:
I believe the name of the optional param will be `waitForWebviewMs `

Reply:
It looks like... this `params` is not necessary? Even when I removed this `params`, the method worked, actually the command name was necessary.